app/src/main/java/net/sourceforge/opencamera/ui/RecSyncView.java

```diff
@@ -61,87 +74,71 @@ public RecSyncView(Context context) {
             Log.d(TAG, "small_screen: " + small_screen);
         }
 
-        final Preview preview = main_activity.getPreview();
+        button_sync_settings = addButton(R.string.sync_settings_unlocked, view -> {
+            if (MyDebug.LOG) {
+                Log.d(TAG, "clicked to buttonSyncSettings");
+            }
 
-        // TODO if (support recSync)
+            main_activity.clickedSyncSettings();
+        });
+
+        button_sync_devices = addButton(R.string.align_phases, view -> {
+            if (MyDebug.LOG) {
+                Log.d(TAG, "clicked to buttonAlignPhases");
+            }
+
+            main_activity.clickedAlignPhases();
+        });
+    }
+
+    public int getTotalWidth() {
+        return (int) (total_width_dp * scale + 0.5f); // convert dps to pixels;
+    }
+
+    public void updateSyncSettingsButton() {
+        button_sync_settings.setText(
+                software_sync_controller.isSettingsBroadcasting() ? R.string.sync_settings_locked : R.string.sync_settings_unlocked
```

Comment:
it has to be checked outside of view and then call view.updateButton(String result)